src/storage/src/hummock/sstable_store.rs

```diff
@@ -196,6 +241,39 @@ impl SstableStore {
         }
     }
 
+    pub async fn get_block_stream(
+        &self,
+        sst: &Sstable,
+        start_block_index: u64,
+    ) -> HummockResult<BlockStream> {
+        // TODO: if sst has owned blocks, start from `max(start_block_idx,
+        // first_missing_sst_block_idx)`
+        let block_loc = if start_block_index != 0 {
+            let block_meta = sst
+                .meta
+                .block_metas
+                .get(start_block_index as usize)
+                .ok_or_else(HummockError::invalid_block)?;
+            let block_loc = BlockLocation {
+                offset: block_meta.offset as usize,
+                size: block_meta.len as usize,
```

Comment:
I think `size` should be the length of the sum of block length after starting index?

Reply:
Fixed and added a unit test.